Cliqz/Extensions/AppDelegateExtension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AppDelegateExtension.swift
+//  Client
+//
+//  Created by Tim Palade on 5/25/18.
+//  Copyright © 2018 Cliqz. All rights reserved.
+//
+
+import UIKit
+
+let InstallDateKey = "InstallDateKey"
+#if GHOSTERY
+let HasRunBeforeKey = "previous_version"
+#endif
+extension AppDelegate {
+    func recordInstallDateIfNecessary() {
+        guard let profile = self.profile else { return }
+        if profile.prefs.stringForKey(LatestAppVersionProfileKey)?.components(separatedBy: ".").first == nil {
+            // Clean install, record install date
+            if UserDefaults.standard.value(forKey: InstallDateKey) == nil {
```

Comment:
you could use `LocalDataStore` or `SettingsPrefs` to store this value

Reply:
LocalDataStore uses UserDefaults. It is the same thing.

Reply:
That is why it is better to use the util classes we have instead of duplicating the code, note that `LocalDataStore` ensures to update `userDefaults` on main thread before there is a firefox code listen to changes to `userDefaults` and it was crashing before if is was called from background thread.

Reply:
That crash happened because fIrefox was listening to the key you were modifying. The key I am using is new, not used by firefox. And on top of that UserDefaults is thread safe...It probably crashed because firefox was listening on the main thread. I don't want to use LocalDataStore as it has dispatching, overhead that I do not need.

Reply:
the crash happened when we were changing our own key and was not related to firefox, that is why we added the dispatch 
regarding the overhead it is only on writing and this will happen only once in your case.

Reply:
The crash should not have happened, ideally. Anyway, I will change this to LocalDataStore.